eclair-core/src/main/scala/fr/acinq/eclair/channel/fund/InteractiveTxBuilder.scala

```diff
@@ -112,6 +112,7 @@ object InteractiveTxBuilder {
                                  lockTime: Long,
                                  dustLimit: Satoshi,
                                  targetFeerate: FeeratePerKw,
+                                 minDepth_opt: Option[Long],
```

Comment:
That was my first approach, but it was awkward since we build the `DualFundedUnconfirmedFundingTx` object in the `InteractiveTxBuilder` approach, where we cannot easily compute the value. I agree it's a little bit hacky, but quite convenient.